libs/vscode/nx-help-and-feedback-view/src/lib/nx-help-and-feedback-provider.ts

```diff
@@ -42,4 +62,50 @@ export class NxHelpAndFeedbackProvider extends AbstractTreeProvider<NxHelpAndFee
       ([title, link, icon]) => new NxHelpAndFeedbackTreeItem(title, link, icon)
     );
   }
+
+  async isConnectedToCloud(): Promise<boolean> {
+    const nxJson = (await readJsonFile(
+      join(getWorkspacePath(), 'nx.json')
+    )) as NxJsonConfiguration;
```

Comment:
Use the `nxWorkspace` function to get this information. Right now the nx specific data isn't exposed, but you can get it from here:
https://github.com/nrwl/nx-console/blob/21c5cc5246c6e46c0e959d4346ef93105972a30b/libs/shared/workspace/src/lib/workspace.ts#L95-L100

and return it somewhere here:
https://github.com/nrwl/nx-console/blob/21c5cc5246c6e46c0e959d4346ef93105972a30b/libs/shared/workspace/src/lib/workspace.ts#L103-L120

Reply:
Seems like the info I need is already returned in the `workspace` property...